spec/calendar.html

```diff
@@ -0,0 +1,153 @@
+      <p>The `from` method takes one argument: item. The following steps are taken:</p>
+      <emu-alg>
+        1. If _item_ has an [[InitializedTemporalCalendar]] internal slot, then
+          1. Return _item_.
+        1. If Type(_item_) is Object and _item_ is not *null*, then
+          1. Return _item_.
+        1. Let _stringIdent_ be ? ToString(_item_).
+        1. Return ? GetBuiltinCalendar(_stringIdent_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-calendar-internal-slots">
+      <h1>Internal Slots</h1>
+
+      <p>The value of the [[BuiltinCalendars]] internal slot must be a record with keys corresponding to each supported calendar system and their corresponding values should be constructor objects that construct a Temporal.Calendar object corresponding to the calendar system.</p>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-the-temporal-calendar-prototype-object">
+    <h1>Properties of the Temporal.Calendar Prototype Object</h1>
+
+    <p>The Temporal.Calendar prototype object</p>
+    <ul>
+      <li>is the intrinsic object %Temporal.Calendar.prototype%.
+      <li>is itself an ordinary object.
+      <li>is not a Temporal.Calendar instance and does not have a [[InitializedTemporalCalendar]] internal slot.
+      <li>has a [[Prototype]] internal slot whose value is %Object.prototype%.
+    </ul>
+
+    <emu-clause id="sec-temporal.calendar.prototype.constructor">
+      <h1>Temporal.Calendar.prototype.constructor</h1>
+      <p>The initial value of Temporal.Calendar.prototype.constructor is %Temporal.Calendar%.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.calendar.prototype-@@tostringtag">
+      <h1>Temporal.Calendar.prototype[ @@toStringTag ]</h1>
+      <p>
+        The initial value of the @@toStringTag property is the string value `"Temporal.Calendar"`.
+      </p>
+      <p>
+        This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-get-temporal.calendar.prototype.id">
+      <h1>get Temporal.Date.prototype.id</h1>
+      <p>
+        `Temporal.Calendar.prototype.id` is an accessor property whose set accessor function is undefined.
+        Its get accessor function performs the following steps:
+      </p>
+      <emu-alg>
+        1. Let _calendar_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
+        1. Return _calendar_.[[ID]].
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.calendar.prototype.tostring">
+      <h1>Temporal.Calendar.prototype.toString ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _calendar_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalDate]]).
+        1. Return _calendar_.[[ID]].
+      </emu-alg>
+    </emu-clause>
```

Comment:
Do we want to define one in terms of the other, so custom calendars can implement just that one? I have a wip change to that effect for TimeZone in <https://github.com/tc39/proposal-temporal/compare/timezone-name>. I don't think it's super useful, but consistency is nice.